__tests__/components/Comments/WriteComment.test.js

```diff
@@ -33,19 +34,15 @@ describe('WriteComment should run correctly', () => {
   it('styles should run correctly', () => {
     expect(styles.replyToContainer).toEqual({
       marginLeft: 60,
-      fontFamily: fonts.inter[600],
-      marginBottom: 11,
-      marginTop: 7,
-      lineHeight: 15,
-      fontSize: 12,
-      color: COLORS.blackgrey
+      alignItems: 'center',
+      flexDirection: 'row'
     });
     expect(styles.container).toEqual({
       flex: 1,
       backgroundColor: COLORS.white,
       width: '100%',
-      paddingRight: 20,
-      paddingLeft: 20,
+      paddingRight: dimen.normalizeDimen(16),
+      paddingLeft: dimen.normalizeDimen(16),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for maintaining consistency in the codebase. The padding values are now being set using the `dimen.normalizeDimen()` function, which ensures that these values are normalized and consistent across different screen sizes.
```diff
- paddingRight: 20,
- paddingLeft: 20,
+ paddingRight: dimen.normalizeDimen(16),
+ paddingLeft: dimen.normalizeDimen(16),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->